testsuites/CBLTester/topology_specific_tests/sync_gateway_proxy_enabled/conftest.py

```diff
@@ -253,6 +256,11 @@ def params_from_base_suite_setup(request):
         expected_sync_gateway_version=sync_gateway_version
     )
 
+    need_sgw_admin_auth = (not disable_admin_auth) and sync_gateway_version >= "3.0"
+    print("==============================================")
```

Comment:
Clean up prints

Reply:
cleaned up the print statement, and added proper log_info